core/arch/arm/plat-imx/conf.mk

```diff
@@ -48,6 +48,7 @@ else ifneq (,$(filter $(PLATFORM_FLAVOR),$(mx6ull-flavorlist)))
 $(call force,CFG_MX6,y)
 $(call force,CFG_MX6ULL,y)
 $(call force,CFG_TEE_CORE_NB_CORE,1)
+$(call force,CFG_NXP_CAAM,n)
```

Comment:
"NXP" -> "IMX"? or this driver will also support LS chips?

Reply:
Yes this driver will support LS chips also.

Reply:
@MrVan ,
 yes the driver will support LS devices as well.